utils/api/requests.js

```diff
@@ -3,9 +3,15 @@ import { fetcher } from './fetcher'
 
 export const getAllRequests = () => {
   const { data, error } = useSWR(`/quote_groups/mine.json`, fetcher)
+  // TODO: this hard codes a quote group ID for now, but it will need to dynamically receive the id from.. somewhere?
```

Comment:
we can get the id from the query params on the request page and pass it in to the appropriate function. (which is not this one btw. 🙃 )

we're doing that in the `getOneWare` function if you wanna take a look there. the id is passed from the "services/[ware].js" page.